### DISCUSSION TITLE
@valibot scope on jsr

### DISCUSSION THREAD

**Discussion by paoloricciuti** [open-circle/valibot#466] [General]
Hey @fabian-hiller given I got access to jsr I registered the valibot scope so that it cannot be registered by someone with bad intentions. Hit me up whenever you register so that I can pass you the ownership 😉

**Comment by fabian-hiller** [comment]
Thank you! I will get back to you once the JSR team has increased my scope limit.

**Comment by fabian-hiller** [comment]
Feel free to pass me the ownership now.

**Reply by paoloricciuti** [reply]
> Feel free to pass me the ownership now.

Invited...as soon as you accept I'll remove myself and make you an admin 😉

**Reply by fabian-hiller** [reply]
Thank you!

**Reply by paoloricciuti** [reply]
Done!

**Reply by fabian-hiller** [reply]
Will publish the library via the scope in the next days 🙌
